include/aikido/control/KinematicSimulationTrajectoryExecutor.hpp

```diff
@@ -29,10 +29,12 @@ class KinematicSimulationTrajectoryExecutor : public TrajectoryExecutor
   /// \param traj Trajectory to be executed. Its StateSpace should be a
   ///        MetaStateSpace over MetaSkeleton, and the dofs in the metaskeleton
   ///        should be all in skeleton passed to the constructor.
+  /// \param skip Whether to skip to the end of the trajectory
```

Comment:
I think this belongs in a different _implementation_ of the `TrajectoryExecutor` interface, not in _each implementation_ of that interface. E.g. we should define a `InstantaneousTrajectoryExeuctor` (or a similar name) that implements `execute()` by jumping to the end of the trajectory.

Reply:
Maybe I'm misunderstanding this, but how do the concrete classes utilize both `TrajectoryExecutor` and `InstantaneousTrajectoryExecutor`? Don't we typically want to have one `Executor` but sometimes just want to jump to the end? e.g. How would it work with `magi.jump()` method?